src/bar.jl

```diff
@@ -47,32 +47,34 @@ grouped_xy(y::AbstractArray) = 1:size(y,1), y
         xmat
     end
 
-    # compute fillrange
-    y, fr = isstack ? groupedbar_fillrange(y) : (y, get(plotattributes, :fillrange, nothing))
-    fillrange := if isylog
-        fill_bottom = min(minimum(y) / 100, the_ylims[1])
-        if isstack
-            # to fill the blank of the bottom segment depeding on if ylims
-            replace!(fr, 0 => fill_bottom)
+    fill_bottom = if isylog
+        if isfinite(the_ylims[1])
+            min(minimum(y) / 100, the_ylims[1])
         else
-            fill_bottom
+            minimum(y) / 100
         end
     else
-        get(plotattributes, :fillrange, nothing)
+        0
+    end
+    # compute fillrange
+    y, fr = isstack ? groupedbar_fillrange(y) : (y, get(plotattributes, :fillrange, [fill_bottom]))
+    if isylog
+        replace!( fr, 0 => fill_bottom )
     end
+    fillrange := fr
 
     seriestype := :bar
     x, y
 end
 
-function groupedbar_fillrange(y)
+function groupedbar_fillrange(y; y_bottom = 0)
     nr, nc = size(y)
     # bar series fills from y[nr, nc] to fr[nr, nc], y .>= fr
     fr = zeros(nr, nc)
     y = copy(y)
     y[.!isfinite.(y)] .= 0
     for r = 1:nr
-        y_neg = 0
+        y_neg = y_bottom
```

Comment:
I don't think this makes sense. This only offsets all negative values by `+y_bottom`. Can't we leave `groupedbar_fillrange` as it was? Isn't this covered by L62 already? Or, respectively, would we even support negative input for log scales?

Reply:
you are right, thats a leftover, I reverted this in https://github.com/JuliaPlots/StatsPlots.jl/commit/bcc35353482e596e32808140ca1257b56164b55c